CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix cluster deletion issues in AWS using `DependsOn`.
 - Fix calico-policy only metrics endpoint.
 - Fix race condition in IPAM locking when lock already acquired.
+- Fix failing go template rendering of KMS encryption content.
```

Comment:
This should go under `Unreleased` I think.

Reply:
fixed